controllers/index.js

```diff
@@ -74,23 +70,38 @@ router.get('/callback', function(req, res) {
       json: true
     };
 
+    //request to get the actual token
     request.post(authOptions, function(error, response, body) {
       if (!error && response.statusCode === 200) {
 
         var access_token = body.access_token,
             refresh_token = body.refresh_token;
 
-        // var options = {
-        //   url: 'https://api.spotify.com/v1/me',
-        //   headers: { 'Authorization': 'Bearer ' + access_token },
-        //   json: true
-        // };
+        var options = {
+          url: 'https://api.spotify.com/v1/me',
+          headers: { 'Authorization': 'Bearer ' + access_token },
+          json: true
+        };
+
+        // use the access token to access the Spotify API
+        request.get(options, function(error, response, body) {
```

Comment:
this function, getting the list of spotify playlists should probably be defined in another file, and 'require'd into this file so you can call the function here